src/components/KSelect/KSelect.vue

```diff
@@ -213,13 +235,8 @@ import KToggle from '@/components/KToggle'
 import KSelectItems from '@/components/KSelect/KSelectItems.vue'
 import KSelectItem from '@/components/KSelect/KSelectItem.vue'
 
-const { getSizeFromString } = useUtilities()
-
-const defaultKPopAttributes = {
-  popoverClasses: 'k-select-popover mt-0',
-  popoverTimeout: 0,
-  placement: 'bottomStart',
-  hideCaret: true,
+export default {
+  inheritAttrs: false,
```

Comment:
issue: This addition may be a breaking change since now classes and other attributes won't be passed in. What's problem does this change address?

Reply:
We had `inheritAttrs` set to `false` in `defineComponent`, this doesn't change it